src/test/java/ModelTest/ItemListTest.java

```diff
@@ -19,32 +19,32 @@ public void setUp(){
         instructor1 = new Instructor("John Smith");
     }
 
-    @org.junit.Test
-    public void addItemToList(){
-        ItemList list = new ItemList("Name", instructor1);
-        list.addItem(item1);
-        assertEquals(1, list.getSize());
-        list.addItem(item2);
-        assertEquals(2, list.getSize());
-    }
-
-    @org.junit.Test
-    public void removePersonFromList(){
-        ItemList list = new ItemList("Name", instructor1);
-        list.addItem(item1);
-        list.addItem(item2);
-
-        list.removeItem(item1);
-        assertEquals(1, list.getSize());
-        list.removeItem(item2);
-        assertEquals(0, list.getSize());
-    }
-
-    @org.junit.Test
-    public void instructorHasItemList(){
-        ItemList list = new ItemList("Name", instructor1);
-        assertTrue(instructor1.getItemList().equals(list));
-    }
+//    @org.junit.Test
+//    public void addItemToList(){
+//        ItemList list = new ItemList("Name", instructor1);
+//        list.addItem(item1);
+//        assertEquals(1, list.getSize());
+//        list.addItem(item2);
+//        assertEquals(2, list.getSize());
+//    }
+//
+//    @org.junit.Test
+//    public void removePersonFromList(){
+//        ItemList list = new ItemList("Name", instructor1);
+//        list.addItem(item1);
+//        list.addItem(item2);
+//
+//        list.removeItem(item1);
+//        assertEquals(1, list.getSize());
+//        list.removeItem(item2);
+//        assertEquals(0, list.getSize());
+//    }
+//
+//    @org.junit.Test
+//    public void instructorHasItemList(){
+//        ItemList list = new ItemList("Name", instructor1);
+//        assertTrue(instructor1.getItemList().equals(list));
+//    }
```

Comment:
just delete it, we have version tracking

```suggestion

```

Reply:
It was already deleted 3 days ago